library/core/src/intrinsics/simd.rs

```diff
@@ -612,6 +612,16 @@ extern "rust-intrinsic" {
     #[rustc_nounwind]
     pub fn simd_fma<T>(x: T, y: T, z: T) -> T;
 
+    /// Computes `(x*y) + z` for each element, with unspecified rounding.
+    ///
+    /// This may be equivalent to `simd_fma`, or it may relax to rounding each
+    /// operation if that's more efficient.
+    ///
+    /// `T` must be a vector of floats.
+    #[cfg(not(bootstrap))]
+    #[rustc_nounwind]
+    pub fn simd_relaxed_fma<T>(x: T, y: T, z: T) -> T;
```

Comment:
Ah I didn't realize there was a scalar version... is it used anywhere?

Reply:
to match the scalar version, imo it should be renamed to `simd_fmuladd`

Reply:
It's not used yet, it was added in preparation for exposing corresponding methods on the float types.

Reply:
> to match the scalar version, imo it should be renamed to simd_fmuladd, also to avoid confusion with any fast-math semantics

That's a pretty bad name though IMO, it is used for the scalar version only because that's how LLVM calls them.

I like `relaxed_fma`. I don't think it is confusing with fast-math semantics, we don't call those "relaxed" after all.

Reply:
hmm, maybe...though I want to say I've seen `relaxed` suggested for fast math functions...

Reply:
though, since it isn't always fused, I think <code>simd_<i>something</i>\_mul_add</code> is better than <code>simd_<i>something</i>\_fma</code>

Reply:
`simd_mul_add`? `simd_relaxed_mul_add`?

Reply:
yeah, either of those would be ok-ish -- `simd_mul_add` makes one think of `f32::mul_add` which is always fused (imo naming `f32::mul_add` `mul_add` is a mistake, but there's nothing we can do now...).